lib/elsa/topic.ex

```diff
@@ -13,6 +13,15 @@ defmodule Elsa.Topic do
     end)
   end
 
+  def exists?(endpoints, topic) do
```

Comment:
👍

Reply:
We should also put a delegate in elsa.ex called topic_exists? that references this function.